ci-operator/step-registry/ipi/aws/pre/ipi-aws-pre-chain.yaml

```diff
@@ -3,6 +3,12 @@ chain:
   steps:
   - chain: ipi-conf-aws
   - chain: ipi-install
+  env:
+    - name: COMPUTE_NODE_TYPE
```

Comment:
This should go [on the `*-ref.yaml` step][1], not the chain.

[1]: https://github.com/openshift/release/pull/13712#discussion_r526040015

Reply:
@wking better?